src/users/user.entity.ts

```diff
@@ -0,0 +1,38 @@
+import { Exclude } from 'class-transformer';
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class User {
+  /**
+   * The unique identifier of the user.
+   * @example 1
+   */
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  /**
+   * The user name.
+   * @example 'Cristiano Ronaldo dos Santos Aveiro'
+   */
+  @Column()
+  name: string;
+
+  /**
+   * The user e-mail.
+   * @example 'cr7@gmail.com'
+   */
+  @Column()
```

Comment:
```suggestion
  @Column({ unique: true })
```

We should also mark the email column as unique to make sure users are not created with the same email.